doc/generate_cet.py

```diff
@@ -36,8 +36,11 @@ def main():
                     cf.write("\n")
                     continue
 
+                
                 topic_name = topic.find('EFDB_Topic').text
                 short_name = topic_name.split("_")[-1]
+                # short_name = '_'.join(split_short_name)
```

Comment:
It would be safer to specify the known number of splits (2 for commands and events, 1 for telemetry) in your call to `split`. That way a non-compliant name with an underscore would be handled correctly.

Please don't commented-out code (especially with no explanation)

Reply:
It works, thanks Russell. Removed commented out code.